stable/velero/templates/deployment.yaml

```diff
@@ -91,6 +91,12 @@ spec:
               value: /credentials/cloud
             - name: VELERO_SCRATCH_DIR
               value: /scratch
+            - name: VELERO_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.namespace
+
```

Comment:
is the empty line necessary here?

Reply:
Nope, thanks for pointing it out. That caused me to realize I had placed it inside the `useSecret` conditional, which isn't correct :)

Reply:
Yeah good catch - I missed that!